aisdc/attacks/multiple_attacks.py

```diff
@@ -0,0 +1,203 @@
+        """
+        with open(self.config_filename, "r+", encoding="utf-8") as f:
+            file_contents = f.read()
+            if file_contents != "":
+                config_file_data = json.loads(file_contents)
+                for config_obj in config_file_data:
+                    params = config_file_data[config_obj]
+                    attack_name = config_obj.split("-")[0]
+                    attack_obj = None
+                    if attack_name == "worst_case":
+                        attack_obj = WorstCaseAttack(**params)
+                        attack_obj.attack(target)
+                    elif attack_name == "lira":
+                        attack_obj = LIRAAttack(**params)
+                        attack_obj.attack(target)
+                    elif attack_name == "attribute":
+                        attack_obj = AttributeAttack(**params)
+                        attack_obj.attack(target)
+
+                    if self.output_filename is not None and attack_obj is not None:
+                        g = GenerateJSONModule(self.output_filename)
+                        _ = attack_obj.make_report(g)
+
+
+class ConfigFile:  # pylint: disable = too-few-public-methods
+    """
+    Module that creates a single JSON configuration file
+    """
+
+    def __init__(
+        self,
+        filename: str = None,
+    ) -> None:
+        self.filename = filename
+
+        dirname = os.path.normpath(os.path.dirname(self.filename))
+        os.makedirs(dirname, exist_ok=True)
+        # if file doesn't exist, create it
+        with open(self.filename, "w", encoding="utf-8") as f:
+            f.write("")
+
+    def add_config(self, config_obj: Any, config_attack_type: str) -> None:
+        """Add a section of JSON to the file which is already open"""
+
+        # Read the contents of the file and then clear the file
+        config_file_data, _ = self.read_config_file()
+
+        # Add the new JSON to the JSON that was in the file, and re-write
+        with open(self.filename, "w", encoding="utf-8") as f:
+            class_name = config_attack_type + "-" + str(uuid.uuid4())
+
+            if isinstance(config_obj, dict):
+                config_file_data[class_name] = config_obj
+            elif isinstance(config_obj, str):
+                with open(str(config_obj), encoding="utf-8") as fr:
+                    config_file_data[class_name] = json.loads(fr.read())
+
+            f.write(json.dumps(config_file_data))
+
+    def read_config_file(self) -> tuple[dict, int]:
+        """Reads a JSON configuration file and returns dictionary
+        with a number of configuration objects"""
+        with open(self.filename, encoding="utf-8") as f:
+            file_contents = f.read()
+            if file_contents != "":
+                config_file_data = json.loads(file_contents)
+            else:
+                config_file_data = {}
+        return config_file_data, len(config_file_data)
```

Comment:
Is there a reason this needs to return two parameters? I can see that the only place the second parameter is used is in the tests, when you check n == 7, but you could change the tests to len(config_file_data) == 7

Unless there's a reason it has to return both parameters obviously

Reply:
We added seven attack configurations, so that is why testing to make sure that there are 7 attack configurations